dashboard/src/shared/url.ts

```diff
@@ -3,9 +3,15 @@ import { IChartVersion, IRepo } from "./types";
 
 export const app = {
   apps: {
-    new: (cluster: string, namespace: string, cv: IChartVersion, version: string) => {
+    new: (
+      cluster: string,
+      namespace: string,
+      cv: IChartVersion,
+      version: string,
+      globalNamespace: string,
```

Comment:
this change (adding as a parameter the `globalNamespace`) is necessary, because when using the `kubeapps` namespace, the link was rendered as non global: e.g. `/c/default/ns/kubeapps/apps/new/bitnami/apache`. When changing the namespace in that scenario, just the namespace gets changed in the URL so it would turn to `/c/default/ns/foo/apps/new/bitnami/apache`, rather than `/c/default/ns/foo/apps/new-from-global/bitnami/apache` which is the correct link.

Reply:
Right - good find.